src/components/bulk-email-tool/bulk-email-form/test/BulkEmailForm.test.jsx

```diff
@@ -2,8 +2,9 @@
  * @jest-environment jsdom
  */
 import React from 'react';
+import { fireEvent } from '@testing-library/react';
 import {
-  render, screen, fireEvent, cleanup, act,
+  render, screen, cleanup, act,
```

Comment:
Is there a reason to use this fireEvent over the one from setupTest? Asking only because I'm wondering if I should follow this in my upcoming PR as well.

Reply:
No, this is actually a IDE screwup, it must have re-imported by accident. Its _should_ be imported from `setupTest.js`